README.md

```diff
@@ -11,11 +11,12 @@ the post-start hooks which are doing the whole magic.
 
 * Install Docker and DDEV Local (and on Windows also Git)
 * Download and extract [this repository](https://github.com/GsTYPO3/ddev-typo3-src/archive/master.zip)
+* [optional] Edit [installation-folder]/.ddev/config.yaml to your likings
```

Comment:
```suggestion
* [optional] Edit `.ddev/config.yaml` to your likings
```

Reply:
@rowild do you think then `[installation-folder]` is important here?

Reply:
I would mention it, since 
- up to this point the user did not switch to the installation folder; and
- `.ddev` is a hidden folder